src/pages/Transfer/CrossChainTransferForm/index.tsx

```diff
@@ -70,6 +68,20 @@ const CrossChainTransferForm = (): JSX.Element => {
             />
           </div>
           <AccountInput account={selectedAccount} accounts={accounts} />
+          <StyledDl direction='column' gap='spacing2'>
```

Comment:
This has been copied from the AMM code—we might want to consider adding `Fees` (or whatever) as a business component.

Reply:
yeah, I agree 100% with you. Do you want me to create a issue for this?